### ISSUE TITLE
h3ToGeoBoundary returns duplicate points for some hexagons

### ISSUE THREAD

**Issue by nrabinowitz** [uber/h3#45]
Certain hexagons (including at least some on the edge of the icosahedron face) have duplicate points generated by `h3ToGeoBoundary`:

```
$ ./bin/h3ToGeoBoundary
894cc536537ffff
894cc536537ffff
{
   18.043333154 293.721634765
   18.042238363 293.720709372
   18.040818259 293.721458061
   18.040492975 293.723132133
   18.041040385 293.723594817
   18.041757122 293.724032885
   18.043007860 293.723308818
   18.043007860 293.723308818
}
```
The last two vertices here are duplicates. The repeated vertex causes problems in the generation of GeoJSON boundaries.

Will update this issue when I find more details on which hexagons are causing the problem.

**Comment by nrabinowitz** [comment]
Pretty sure this is the result of some Class III hexagons having vertexes directly on the icosahedron edge - those vertices are duplicated.

**Comment by isaacbrodsky** [comment]
We noticed that these duplicated vertices are shared with the parent class II hexagon, which might be useful when excluding them from output.

**Comment by nrabinowitz** [comment]
Closed in #46 .
